lib/Doctrine/DBAL/Connection.php

```diff
@@ -2085,7 +2085,7 @@ private function _bindTypedValues($stmt, array $params, array $types)
      *
      * @return mixed[] [0] => the (escaped) value, [1] => the binding type.
```

Comment:
Let's improve the phpdoc as well?
```suggestion
     * @return array{mixed, mixed} [0] => the (escaped) value, [1] => the binding type.
```

Reply:
Yeah, why not. The second component of the tuple has to be an `int` actually.